config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
+
+  get '/trip_dashboard', to: "trips/trips#new"
+
```

Comment:
This might be a dumb question, but why is this trips/trips#new instead of just trips#new?